pallets/multisig/src/benchmarking.rs

```diff
@@ -0,0 +1,267 @@
+// This file is part of the Polymesh distribution (https://github.com/PolymathNetwork/Polymesh).
+// Copyright (c) 2020 Polymath
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(feature = "runtime-benchmarks")]
+use crate::*;
+use frame_benchmarking::benchmarks;
+use frame_system::RawOrigin;
+use pallet_identity::benchmarking::make_account;
+
+pub type MultiSig<T> = crate::Module<T>;
+pub type Identity<T> = identity::Module<T>;
+pub type Signers<AccountId> = Vec<(AccountId, RawOrigin<AccountId>, IdentityId)>;
+
+fn generate_signers<T: Trait>(signers: &mut Signers<T::AccountId>, n: usize) {
+    for x in 0..n {
+        signers.push(make_account::<T>("key", x as u32));
```

Comment:
As we discussed earlier, `make_account` registers an identity for the key. Multisig signers should not have an identity. You can add a new `make_account_without_did` next to the `make_account` function that looks something like
```
fn make_account_without_did<T: Trait>(name: &'static str, u: u32) -> (T::AccountId, RawOrigin<T::AccountId>) {
    let account_id: T::AccountId = account(name, u, SEED);
    let origin = RawOrigin::Signed(account_id.clone());
    let _ = balances::Module::<T>::make_free_balance_be(&account_id, 1_000_000.into());
    (account_id, origin)
}
```